front/src/modules/rollingStock/helpers/utils.ts

```diff
@@ -159,6 +160,7 @@ export const rollingStockEditorQueryArg = (
     unit: data.unit,
   },
   effort_curves: currentRsEffortCurve,
+  base_power_class: (data.basePowerClass as string) || '',
```

Comment:
No need for the cast here
```suggestion
  base_power_class: data.basePowerClass || '',
```

Reply:
done